processor/processor.go

```diff
@@ -3000,7 +3000,7 @@ func (proc *Handle) handlePendingGatewayJobs(partition string) bool {
 		return false
 	}
 
-	rsourcesStats := rsources.NewStatsCollector(proc.rsourcesService)
+	rsourcesStats := rsources.NewStatsCollector(proc.rsourcesService, rsources.IgnoreDestinationID())
```

Comment:
`handlePendingGatewayJobs` is only used by tests, shouldn't we plan to remove it and adapt tests?